docs/goals-canvas.md.txt

```diff
@@ -0,0 +1,26 @@
tie/Haal-Centraal-BRP-bevragen/issues/772){:target="_blank"}, [773](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/737){:target="_blank"}, [774](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/774) [774](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/734){:target="_blank"}, [732](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/732){:target="_blank"}, [734](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/734){:target="_blank"}, [736](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/736){:target="_blank"}, [758](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/758){:target="_blank"}, [733](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/733){:target="_blank"}, [729](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/729),  [728](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/728),  [727](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/727){:target="_blank"}, [726](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/726){:target="_blank"}, [725](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/725){:target="_blank"}, [724](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/724){:target="_blank"}, [721](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/721){:target="_blank"}, [717](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/717){:target="_blank"}, [714](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/714){:target="_blank"}, [713](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/713){:target="_blank"}, [715](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/715){:target="_blank"}, [712](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/712){:target="_blank"}, [710](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/710){:target="_blank"}, [767](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/767){:target="_blank"}, [689](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/689),  [675](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/675),   [769](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/769),  [280](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/280), {:target="_blank"}, [279](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/279){:target="_blank"}, [277](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/277){:target="_blank"}, [276](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/276){:target="_blank"}, [275](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/275){:target="_blank"}, [274](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/274){:target="_blank"}, |
```

Comment:
Bij het zoeken krijgt je ook "in onderzoek". Als het hier expliciet wordt opgenomen, zou ik het ook bij zoeken doen.